pygfx/renderers/wgpu/wgsl/text.wgsl

```diff
@@ -237,7 +237,8 @@ fn fs_main(varyings: Varyings) -> FragmentOutput {
     // z-fighting to occur leading to artifacts.
     // See: https://github.com/pygfx/pygfx/pull/776#issuecomment-2149374275
     // And: https://github.com/pygfx/pygfx/pull/774#issuecomment-2147458827
-    out.depth = varyings.position.z - 1e-6 * (2.0 - outline);
+    // float32 precision is about 5-6 digits
+    out.depth = varyings.position.z * (0.99998 + 1e-5 * outline);
```

Comment:
I'm not sure if your intention is to try and take the smallest possible step in float32, but you can directly do that like this:

```
uint asInt = asuint(myFloat);
asInt++;
float next = asfloat(asInt);
```

(pseudocode, I don't actually know how to do this in WGSL)

ignore this comment if it's not what you're trying to do :)

Reply:
I am trying to do that. But I’ve been around floats long enough to know that this isn’t very smart to do since GPUs sometimes don’t adhere to IEEE for floats and as the float gets really small or really big, the standard way floats work isn’t really guaranteed. 

Finally. We might want to increase or decrease the step size in the future.

Reply:
```suggestion
    // The math below is equivalent to (1 - 1e-5 * (2 - outline))
    out.depth = varyings.position.z * (0.99998 + 1e-5 * outline);
```

Reply:
Yeah. Sounds good to me!